.gitignore

```diff
@@ -0,0 +1 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Personally I would consider a user specific `.gitignore` bad practise, who cares about a couple of extra lines in a `.gitignore` file. Not that there aren't use cases for it, but generally speaking.

@schnumsel The current `.gitignore` is effectively empty though, might as well not have it.